ostd/src/task/processor.rs

```diff
@@ -43,7 +43,9 @@ pub(super) fn switch_to_task(next_task: Arc<Task>) {
     let current_task_ctx_ptr = if !current_task_ptr.is_null() {
         // SAFETY: The current task is always alive.
         let current_task = unsafe { &*current_task_ptr };
-        current_task.save_fpu_state();
+        if let Some(fpu_state) = Task::current().unwrap().fpu_state() {
+            fpu_state.borrow_mut().save();
+        }
```

Comment:
Since this is OSTD code, unsafe function can be used, how about just using `Task::fpu_state()` instead of `CurrentTask::fpu_state`? Because `current_task` is constructed in line 45, we can directly visit `current_task..fpu_state()`? It seems more efficient than calling `Task::current()`